sale_generator/views/sale_view.xml

```diff
@@ -1,56 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<odoo>
-    <record id="view_sale_order_form" model="ir.ui.view">
```

Comment:
Not sure it should be removed because it's required to set tmpl_sale_id in generator